dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -60,10 +60,13 @@ recursiveSearch=Perform a nested group search
 recursiveSearch.desc=Performs a nested group search. Select this option only if the LDAP server does not support recursive server-side searches.
 
 searchTimeout=Search timeout
-searchTimeout.desc=Maximum time for an LDAP server to respond before a request is canceled. This is equivalent to a read timeout once the connection is established.
+searchTimeout.desc=Maximum time for an LDAP server to respond before a request is canceled.
 
-connectTimeout=Timeout duration for establishing LDAP connection
-connectTimeout.desc=Maximum time for establishing a connection to the LDAP server. The program logs an error message if the specified time expires.
+connectTimeout=Timeout for establishing LDAP connection
```

Comment:
Change "Timeout for establishing LDAP connection" to "Timeout for establishing an LDAP connection". (Add "an" and add a period at the end.)

Reply:
I have updated the label wording, but did not add the period as labels should not be sentences. I believe the build fail if I do.